src/view/uniprotkb/components/ProteinNamesView.tsx

```diff
@@ -136,17 +131,22 @@ const getInfoListForNames = (name: ProteinNames): {title: string; content: JSX.E
       content: (
         <Fragment>
           {name.shortNames
-            .map((shortName):JSX.Element => <NameWithEvidence data={shortName} key={v1()} />)
-            .reduce((acc, shortName): JSX.Element =>
-              acc === null ? (
-                shortName
-              ) : (
-                <Fragment>
-                  {acc}
-;
-                  {shortName}
-                </Fragment>
+            .map(
+              (shortName): JSX.Element => (
+                <NameWithEvidence data={shortName} key={v1()} />
               )
+            )
+            .reduce(
```

Comment:
Looks like this change is due to whitespace so sorry to be picky at this point. Just wondering if it's possible to combine the `map` and `reduce` into a single `reduce`?